tools/qmlformat.py

```diff
@@ -28,7 +29,12 @@ def find_qt_version():
 
 
 def main(argv=None):
-    qmlformat_executable = shutil.which("qmlformat")
+    # First look up at the most common location for QT6 which is not in PATH
+    qmlformat_executable = shutil.which(
+        "qmlformat", os.F_OK, "/usr/lib/qt6/bin"
```

Comment:
This does not work in Fedora where the path is "/usr/lib64/qt6/bin"

Reply:
The executable is in the PATH and named `qmlformat-qt6`.

These are actually 2 independent copies of the executable:
- `/usr/bin/qmlformat-qt6`
- `/usr/lib64/qt6/bin/qmlformat`

Reply:
And also at "/usr/lib/qt6/bin" depending on the architecture: https://fedora.pkgs.org/34/fedora-updates-x86_64/qt6-qtdeclarative-devel-6.2.2-1.fc34.i686.rpm.html

I will add `/usr/bin/qmlformat-qt6` to the lookup paths.

Reply:
32-bit is no longer supported.